integreat_cms/cms/models/pages/page_translation.py

```diff
@@ -100,8 +100,37 @@ def short_url(self):
             "public:expand_page_translation_id", kwargs={"short_url_id": self.id}
         )
 
+    @cached_property
+    def mirrored_translation_text_internal(self):
+        """
+        This method returns the text of the mirrored translation or None,
+        if it is not available in this language
+
+        :return: The text or None
+        :rtype: str or None
+        """
+        if not self.page.mirrored_page:
+            return None
+
+        translation = self.page.get_mirrored_page_translation(self.language.slug)
+        if translation and translation.content:
+            return translation.content
+
+        return None
```

Comment:
```suggestion
        if self.page.mirrored_page:
            if translation := self.page.get_mirrored_page_translation(self.language.slug):
                return translation.content
        return ""
```
I think this can be simplified a bit